xray/commands/audit/generic/auditmanager.go

```diff
@@ -240,46 +247,49 @@ func doAudit(params *Params) (results []services.ScanResponse, isMultipleRoot bo
 			errorList.WriteString(fmt.Sprintf("'%s' audit command failed:\n%s\n", tech, e.Error()))
 			continue
 		}
+		techResults = audit.BuildImpactPathsForScanResponse(techResults, params.dependencyTrees)
 		results = append(results, techResults...)
 		isMultipleRoot = len(dependencyTrees) > 1
 	}
 	if errorList.Len() > 0 {
-		err = errors.New(errorList.String())
+		err = errorutils.CheckErrorf(errorList.String())
 	}
 	return
 }
 
-func getTechDependencyTree(params *Params, tech coreutils.Technology) (dependencyTrees []*services.GraphNode, e error) {
+func getTechDependencyTree(params *Params, tech coreutils.Technology) (dependencyTrees []*services.GraphNode, err error) {
 	if params.progress != nil {
 		params.progress.SetHeadlineMsg(fmt.Sprintf("Calculating %v dependencies", tech.ToFormal()))
 	}
 	switch tech {
 	case coreutils.Maven, coreutils.Gradle:
-		dependencyTrees, e = getJavaDependencyTree(params, tech)
+		dependencyTrees, err = getJavaDependencyTree(params, tech)
 	case coreutils.Npm:
-		dependencyTrees, e = npm.BuildDependencyTree(params.args)
+		dependencyTrees, err = npm.BuildDependencyTree(params.args)
 	case coreutils.Yarn:
-		dependencyTrees, e = yarn.BuildDependencyTree()
+		dependencyTrees, err = yarn.BuildDependencyTree()
 	case coreutils.Go:
-		dependencyTrees, e = _go.BuildDependencyTree(params.serverDetails, params.depsRepo)
+		dependencyTrees, err = _go.BuildDependencyTree(params.serverDetails, params.depsRepo)
 	case coreutils.Pipenv, coreutils.Pip, coreutils.Poetry:
-		dependencyTrees, e = python.BuildDependencyTree(&python.AuditPython{
+		dependencyTrees, err = python.BuildDependencyTree(&python.AuditPython{
 			Server:              params.serverDetails,
 			Tool:                pythonutils.PythonTool(tech),
 			RemotePypiRepo:      params.depsRepo,
 			PipRequirementsFile: params.requirementsFile})
 	case coreutils.Nuget:
-		dependencyTrees, e = nuget.BuildDependencyTree()
+		dependencyTrees, err = nuget.BuildDependencyTree()
 	default:
-		e = errorutils.CheckError(fmt.Errorf("%s is currently not supported", string(tech)))
+		err = errorutils.CheckError(fmt.Errorf("%s is currently not supported", string(tech)))
```

Comment:
`errorutils.CheckError(fmt.Errorf` can be simplified to `errorutils.CheckErrorf`